### DISCUSSION TITLE
Should `preset-commonmark` nodes be defining styles?

### DISCUSSION THREAD

**Discussion by tantaman** [Milkdown/milkdown#346] [Ideas]
I wanted to change the margins around headings.

I started doing this through creating a new theme but found that my styles were not being applied.

Digging further, it looks like the `preset-commonmark` node types define their own styles. See here:
https://cs.github.com/Saul-Mirone/milkdown/blob/6d23137941cc22e5f80459819bda850e3411019c/packages/preset-commonmark/src/node/heading.ts#L66

I think it might make more sense to remove styling from those core plugins and instead rely on the theme to provide the styles.

**Comment by Saul-Mirone** [comment]
I'll try to gather the styles into theme, it's on my plan. But in fact I've already tried once before. The problem is that if we put all things into theme. Write a theme can be really hard because it needs to take care of all the plugins. Such as tooltip and slash. I'm still finding a proper way to do this.

**Comment by Saul-Mirone** [comment]
WIP here:
https://github.com/Saul-Mirone/milkdown/pull/347
